src/main/java/ch/njol/skript/hooks/spark/expressions/ExprMSPT.java

```diff
@@ -0,0 +1,94 @@
+package ch.njol.skript.hooks.spark.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.hooks.spark.SparkHook;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import me.lucko.spark.api.Spark;
+import me.lucko.spark.api.SparkProvider;
+import me.lucko.spark.api.statistic.StatisticWindow;
+import me.lucko.spark.api.statistic.misc.DoubleAverageInfo;
+import me.lucko.spark.api.statistic.types.GenericStatistic;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+public class ExprMSPT extends SimpleExpression<Number> {
+
+	private int index;
+	private String expr = "mspt";
+
+	static {
+		Skript.registerExpression(ExprMSPT.class, Number.class, ExpressionType.SIMPLE,
+			"[the] server tick [(duration|time] [(within|from) the last 10[ ]s[econds]]",
+			"[the] server tick [(duration|time] [(within|from) the last ([1] minute|1[ ]m[inute])]",
+			"[the] server tick [(duration|time] [(within|from) the last 5[ ]m[inutes]]",
+			"[the] server tick [(duration|time[s]");
```

Comment:
```suggestion
			"[the] server tick [(duration|time)] [(within|from) the last 10[ ]s[econds]]",
			"[the] server tick [(duration|time)] [(within|from) the last ([1] minute|1[ ]m[inute])]",
			"[the] server tick [(duration|time)] [(within|from) the last 5[ ]m[inutes]]",
			"[the] server tick [duration|time[s]]");
```

I also don't get why you are making the choices in the other syntax optional? This goes for ExprCpuUsage too.